responses/cs/HassClimateSetTemperature.yaml

```diff
@@ -0,0 +1,5 @@
+language: cs
+responses:
+  intents:
+    HassClimateSetTemperature:
+      default: Temperature set to {{ slots.temperature }} degrees
```

Comment:
default: 'Teplota nastavena na {{ slots.temperature }} {{ ''stupeň'' if state.state == 1 else ''stupně'' if state.state in [2,3,4] else ''stupňů'' }}'